### ISSUE TITLE
Add more options to authorized_keys modul for commenting/adding notes

### ISSUE THREAD

**Issue by offlinehoster** [ansible/ansible-modules-core#2873]
##### ISSUE TYPE

Bug Report
##### COMPONENT NAME

authorized_key module
##### ANSIBLE VERSION

N/A
##### SUMMARY

It would be nice to have something like

```
# username1
ssh-rsa …

# username2
ssh-rsa …
```

A note for each key, tagged by a name/username/note or whatever.
Adding this information to the pub key file is not possible. It will just add the key itselt without the tagged information.

Cheers
Christian

**Comment by ansibot** [comment]
@ansible ping, this issue is waiting for your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by ansibot** [comment]
This issue was migrated to https://github.com/ansible/ansible/issues/29928
